Corona-Warn-App/src/test/java/de/rki/coronawarnapp/presencetracing/checkins/qrcode/ValidQRCodeProvider.kt

```diff
@@ -91,6 +92,25 @@ class ValidQRCodeProvider : ArgumentsProvider {
                             .build()
                     )
                     .build()
+            ),
+            Arguments.of(
+                TraceLocationOuterClass.QRCodePayload.newBuilder()
+                    .setVersion(1)
+                    .setCrowdNotifierData(
+                        TraceLocationOuterClass.CrowdNotifierData.newBuilder()
+                            .setCryptographicSeed(CRYPTOGRAPHIC_SEED.decodeBase64()!!.toProtoByteString())
+                            .setPublicKey(PUB_KEY.decodeBase64()!!.toProtoByteString())
+                            .setVersion(1)
+                    )
+                    .setVendorData("CAEQARgK".decodeBase64()!!.toProtoByteString())
+                    .setLocationData(
+                        TraceLocationOuterClass.TraceLocation.newBuilder()
+                            .setDescription("Ice Cream Shop with a very long description that has more than one hundred characters as allowed by Luca app.")
+                            .setAddress("Main Street 1")
```

Comment:
Maybe also make the address > 100 characters long

Reply:
done